hil-test/tests/embassy_interrupt_spi_dma.rs

```diff
@@ -117,6 +146,26 @@ mod test {
         .with_dma(dma_channel2.configure(false, DmaPriority::Priority0))
         .into_async();
 
+        #[cfg(not(any(esp32, esp32s2, esp32s3)))]
```

Comment:
I think it'd be fine to use I2s for all devices, although I guess testing 2 SPIs has some value as well.

Reply:
I especially didn't want to use I2S on ESP32 for now since .... it might introduce some flakiness (maybe)
Also, I guess it's not bad to have "mixed" test setups